eclair-core/src/main/scala/fr/acinq/eclair/crypto/keymanager/LocalChannelKeyManager.scala

```diff
@@ -97,12 +99,15 @@ class LocalChannelKeyManager(seed: ByteVector, chainHash: ByteVector32) extends
    * @param publicKey        extended public key
    * @param txOwner          owner of the transaction (local/remote)
    * @param commitmentFormat format of the commitment tx
-   * @return a signature generated with the private key that matches the input
-   *         extended public key
+   * @return a signature generated with the private key that matches the input extended public key
    */
   override def sign(tx: TransactionWithInputInfo, publicKey: ExtendedPublicKey, txOwner: TxOwner, commitmentFormat: CommitmentFormat): ByteVector64 = {
-    val privateKey = privateKeys.get(publicKey.path)
-    Transactions.sign(tx, privateKey.privateKey, txOwner, commitmentFormat)
+    val tags = TagSet.Empty.withTag(Tags.TxOwner, txOwner.toString).withTag(Tags.TxType, Tags.TxTypes.CommitTx)
```

Comment:
```suggestion
    // NB: not all those transactions are actually commit txes (especially during closing), but this is good enough for monitoring purposes
    val tags = TagSet.Empty.withTag(Tags.TxOwner, txOwner.toString).withTag(Tags.TxType, Tags.TxTypes.CommitTx)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1659/commits/11e54785be3e67dc6c2598dbe60c9b45191c3216